rust/src/importer.rs

```diff
@@ -44,6 +62,18 @@ pub fn tweak_imported_file_info(
             file_info.set_attribute_uint32("unix::mode", mode);
         }
     }
+
+    if filetype == FileType::SymbolicLink {
+        if let Some(target) = file_info.symlink_target() {
+            // See above, this is a special case hack until
+            // https://github.com/fedora-sysv/chkconfig/pull/67 propagates everywhere
+            // and/or https://github.com/ostreedev/ostree-rs-ext/pull/182 merges.
+            if target.ends_with("//sbin/chkconfig") {
```

Comment:
What I had in mind was a filter matching on the exact source path (i.e. not the target content), but it should be an irrelevant difference. This works for me too.